### PR TITLE
Hydration errors 2

### PR DESCRIPTION
**Pull request by deetz99**
fixed order of bg images on ppr marketing page causing hydration errors

### CODE REVIEW

File: app/pages/ppr-marketing.vue [hunk 1]
```diff
@@ -73,6 +75,13 @@ const { data: pprSections } = await useAsyncData(`ppr-sections-${locale.value}`,
   return queryContent().where({ _locale: locale.value, _path: { $contains: 'ppr/sections' } }).find()
 })
 
+// srr safe media query
+watchDebounced(width, () => {
```
Comment (thorwolpert) [inline app/pages/ppr-marketing.vue:79]: interesting, for hydration issues?

**Review by thorwolpert — APPROVED** [review]

File: app/pages/ppr-marketing.vue (hunk 1 above)
Comment (deetz99) [reply app/pages/ppr-marketing.vue:79]: Yes, initial render the ref is false on server and true on client so it causes a mismatch. This way the value is the same for server/client and still updates when the screen width changes, used debounced watcher to update once the media query ref changes and to not fire multiple events if the screen changes width after mount.

**Comment by seeker25** [comment]
@deetz99  I've deployed main back to dev.. there are some 404 screens we keep seeing with login and logout

**Comment by seeker25** [comment]
Clicking here for example:

image

**Comment by deetz99** [comment]
@seeker25 

Okay thanks, I'll need to add in the login/logout redirects then. Do you know if the signin/bceid, signin/bcsc and signin/bceid are in use at all ?

**Comment by seeker25** [comment]
Yes they all are

**Comment by deetz99** [comment]
Thank you

**Pull request merged**